test/Mono.Linker.Tests.Cases/Reflection/MethodUsedViaReflection.cs

```diff
@@ -484,10 +436,19 @@ public static int OnlyCalledViaReflection (int foo, int bar, int baz)
 			{
 				return 45;
 			}
+
+			[Kept]
+			[RecognizedReflectionAccessPattern]
+			public static void TestNameWithIntAndBindingFlags ()
+			{
+				var method = typeof (GetMethod_Name_GenericParameterCount_BindingAttr_Binder_Types_Modifiers).GetMethod ("OnlyCalledViaReflection", 1, BindingFlags.Public, null, new Type[] { }, null);
```

Comment:
Can we add another BindingFlag to check that is actually using the BindingFlags?